news/_posts/2020-04-27-Polymake.jl-0.4-has-been-tagged.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Polymake.jl 0.4 has been tagged
+author: Marek Kaluba
+---
+
+`Polymake.jl` v0.4.0 has been tagged!
+
+Since we're upgrading to support `CxxWrap.jl` `v0.10.1`, we decided to bump the major 
+version to force compilation of libpolymake against libcxxwrap-julia. 
+The release has been somewhat hastened by the need for `Oscar.jl` jointly working with 
+the newest `CxxWrap.jl`, so expect `v.0.4.1` and later minor versions rather heavy
+in added features :)
+
+<br/>
+_Note_:
+We're tracking the version of libcxxwrap-julia used, so that we can warn You
+that `import Pkg; Pkg.build("Polymake")` is necessary, but some of the cases
+might have slipped us. If you experience crashes while `using Polymake` try:
+ * `import Pkg; Pkg.free("CxxWrap")`,
+ * `import Pkg; Pkg.build("Polymake")`
+ * filing bug report!
+
+Below you can find a summary of the major changes and design decision we have 
+taken in the development of Polymake.jl.
```

Comment:
```suggestion
taken in the development of `Polymake.jl`.
```
Consistency

Reply:
I was wondering on this one, we are not consistent across the website. I will then update other NEWS accordingly.